app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -54,7 +59,12 @@ const submitWithWorldpay3dsFlexDdcResult = form => {
       const { MessageType, SessionId, Status } = JSON.parse(event.data)
       if (MessageType === 'profile.completed') {
         if (Status === true && typeof SessionId === 'string') {
+          updateDdcStatusField('valid DDC result')
           addWorldpaySessionIdToForm(form, SessionId)
+        } else if (!Status) {
+          updateDdcStatusField('DDC response had Status of false')
+        } else {
+          updateDdcStatusField('SessionID missing in DDC response')
```

Comment:
Technically, this could be it exists but isn’t a string.

Reply:
true, but I reckon logging something that is almost right is fine, I imagine people will be digging into code to determine what this means when its seen anyway

Reply:
I suppose it could say something like “no SessionID string in DDC result” (which leads me on to another thing I’ve just noticed: inconsistent about whether we call it a “result” or a “response”).